src/algebra/lie/cartan_subalgebra.lean

```diff
@@ -55,6 +58,36 @@ lemma le_normalizer_of_ideal {N : lie_subalgebra R L}
   (h : ∀ (x y : L), x ∈ N → y ∈ H → ⁅x,y⁆ ∈ H) : N ≤ H.normalizer :=
 λ x hx y, h x y hx
 
+/-- See also `lie_subalgebra.lt_normalizer_iff_nontrivial`. -/
+lemma normalizer_eq_self_iff_subsingleton :
+  H.normalizer = H ↔ subsingleton (lie_module.max_triv_submodule R H $ L ⧸ H.to_lie_submodule) :=
```

Comment:
Can't you say
```suggestion
  H.normalizer = H ↔ (lie_module.max_triv_submodule R H $ L ⧸ H.to_lie_submodule) = ⊥ :=
```
Would that make more sense? Should we have both?

Reply:
Yes, definitely. What I had was silly. Now fixed!